### PR TITLE
removed false info that breaks gathers

### PR DESCRIPTION
**Pull request by scapps00**
removed text that indicated in postCallsCallIdGather.md that the terminatingDigits parameter can be empty and emphasized that terminatingDigits cannot be set to "". This actually breaks the gather

## For the Committer

- [x] I made sure that the site looks great and functions perfectly live
- [x] I ran splell check
- [x] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-apdocs-samantha-little-fixes.s3-website-us-east-1.amazonaws.com/

File: methods/calls/postCallsCallIdGather.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Collects a series of DTMF digits from a phone call with an optional prompt. This
...
-| terminatingDigits  | A string of DTMF digits that end the gather operation immediately if any one of them is detected (default “\#”; an empty string means collect all DTMF until maxDigits or the timeout).<br>Example:<br>`#` : The gather ends if \# is detected (this is the default behavior if the terminatingDigits property is not specified)<br>`0#*` : The gather ends if either 0, \#, or \* is detected<br>Don’t forget to encode keypad digits that have special meaning in a URL, like \#. | No        |
+| terminatingDigits  | A string of DTMF digits that end the gather operation immediately if any one of them is detected (default “\#”). Cannot be empty. <br>Example:<br>`#` : The gather ends if \# is detected (this is the default behavior if the terminatingDigits property is not specified)<br>`0#*` : The gather ends if either 0, \#, or \* is detected<br>Don’t forget to encode keypad digits that have special meaning in a URL, like \#. | No        |
```
Comment (Aubron) [inline methods/calls/postCallsCallIdGather.md]: If it cannot be empty, should we set it to required? Or is it just that you can't supply it empty.

Recommend "If set, this cannot be empty" if not required, or simply changing the required field to yes and removing this sentence if it is.

**Comment by Aubron** [comment]
Flagging this again for requested changes.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-apdocs-samantha-little-fixes.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-apdocs-samantha-little-fixes.s3-website-us-east-1.amazonaws.com/

**Pull request merged**